### ISSUE TITLE
Empty changelog

### ISSUE THREAD

**Issue by slorber** [lerna/lerna-changelog#354]
Hi,

I'm working on [docusaurus](https://github.com/facebook/docusaurus). We are long-time lerna-changelog users.

I noticed that Lerna gives me unpredictable changelogs. 

A command can give me an empty changelog, and 10 seconds later, the exact same command gives me the changelog I want:

![image](https://user-images.githubusercontent.com/749374/88818438-fef9da00-d1be-11ea-83fe-710f08bdd117.png)

As I'm not able to enable verbose logging or whatever, I don't really know what is happening.

I noticed that when the changelog is empty, the last % of the progress bar takes much longer just before.

Also noticed that running `lerna-changelog --from lastReleaseTag` is much faster than just `lerna-changelog`, and much more likely to give me a non-empty output. Basically, I feel

Has anyone an idea about what is going on?

The repo is public and my token has the correct permissions, afaik:

![image](https://user-images.githubusercontent.com/749374/88818837-762f6e00-d1bf-11ea-939f-5939d7ca4615.png)

**Comment by Turbo87** [comment]
hmm, thanks for reporting this. I haven't noticed any such issues so far unfortunately

> As I'm not able to enable verbose logging or whatever, I don't really know what is happening.

yeah, I guess we could/should introduce something like https://github.com/visionmedia/debug to enable verbose logging if such things happen 🤔

**Comment by slorber** [comment]
Yes that could be helpful.

For example, just logging which tag it uses when I don't provide `--from` attribute would be nice

**Comment by slorber** [comment]
FYI I tried with a new Github token and it didn't work better :'(

**Comment by izelnakri** [comment]
@Turbo87 I have the sample problem, `lerna-changelog` shows the progress bar and then silently fails(succeeds) without creating a CHANGELOG, `lerna-changelog --from lastReleaseTag` doesnt help either, it silently fails on all cases. You can try it here, we should probably make `lerna-changelog` more strict so it never fails silently.


https://github.com/lerna/lerna-changelog

**Comment by Turbo87** [comment]
@izelnakri the PRs in your project don't have any labels, which is why they don't show up in the generated changelog. see https://github.com/lerna/lerna-changelog#usage

**Comment by izelnakri** [comment]
Yes, I just realized that after an hour of debugging :facepalm: Added an explanation PR here: https://github.com/lerna/lerna-changelog/pull/391
